examples/text-classification/run_glue.py

```diff
@@ -215,6 +215,12 @@ def compute_metrics_fn(p: EvalPrediction):
             test_datasets.append(
                 GlueDataset(mnli_mm_data_args, tokenizer=tokenizer, mode="test", cache_dir=model_args.cache_dir)
             )
+            # Before conducting on diagnostic task, diagnostic.tsv should be copied from
```

Comment:
I think this should be in another if case, like:
```
if data_args.task_name == "diagnostic":
```

You can then also remove this comment and load the file directly from `{your_path}/glue_data/diagnostic `